pkg/kine/logstructured/sqllog/sql.go

```diff
@@ -619,6 +620,27 @@ func (s *SQLLog) Append(ctx context.Context, event *server.Event) (int64, error)
 	return rev, nil
 }
 
+func (s *SQLLog) Create(ctx context.Context, key string, value []byte, lease int64) (rev int64, err error) {
+	ctx, span := otelTracer.Start(ctx, fmt.Sprintf("%s.Create", otelName))
+	defer func() {
+		span.RecordError(err)
+		span.SetAttributes(attribute.Int64("revision", rev))
+		span.End()
+	}()
+	span.SetAttributes(attribute.String("key", key))
+
+	rev, err = s.d.Create(ctx, key, value, lease)
+	if err != nil {
+		return 0, err
+	}
+
+	select {
+	case s.notify <- rev:
```

Comment:
This notifies the watcher on the creation of this revision. This is part of the original implementation, we can change it in a future update of the watcher.